core/web/schema/type/chain.graphql

```diff
@@ -11,3 +11,11 @@ type ChainsPayload implements PaginatedPayload {
     results: [Chain!]!
     metadata: PaginationMetadata!
 }
+
+enum Network {
```

Comment:
Do we need to add this enum in, or is there some way to make it dynamic so that when we have new chain types, we don't need to update this

Reply:
Hmm, alternatively we could change the type to string and validate it in the backend which would remove the graphql typing.

Not sure there is a way to generate the graphql enum from golang enum.

Reply:
In the future we will not know the chains ahead of time. We need to be prepared for that.

Reply:
alright , sounds like changing to string might be a better option then